src/Umbraco.Core/Models/Mapping/ContentPropertyMapDefinition.cs

```diff
@@ -42,9 +46,28 @@ public class ContentPropertyMapDefinition : IMapDefinition
             entityService,
             textService,
             loggerFactory.CreateLogger<ContentPropertyDisplayMapper>(),
-            propertyEditors);
+            propertyEditors,
+            dataTypeConfigurationCache);
     }
 
+    [Obsolete]
```

Comment:
Lets add a message to this attribute as well:

```suggestion
    [Obsolete("Please use constructor that takes an IDataTypeConfigurationCache. Will be removed in V14.")]
```